content/en/community/_index.html

```diff
@@ -39,10 +39,62 @@ <h2>Develop and Contribute</h2>
+  <div class="col-xs-5 col-sm-6 col-md-6 col-lg-6" style="margin-top: 3%">
+  <h2>Where to Start?</h2>
+  </p>If you're looking for a place to start for your first contribution, here's a few things to note. The CloudWeGo project is subdivided into subprojects:
+    <ul>
+      <li><p>Kitex (Kitex & Kitex ecosystem & kitex-contrib)
+      <li><p>Hertz (Hertz & Hertz ecosystem & hertz-contrib)
+      <li><p>Volo (Volo & Volo ecosystem & volo-rs & Motore & Pilota)
+      <li><p>Netpoll (Netpoll & Netpoll ecosystem)
+      <li><p>Monoio (Monoio & Monoio ecosystem)
+      <li><p>Serdes (Thriftgo & Frugal & Fastpb & Sonic & thrift-gen-validator)
+      <li><p>Shmipc (shmipc-spec & shmipc-go)
+      <li><p>Website & Docs (cloudwego.github.io & community)
+The community is structured with various roles to streamline day to day operations, and to recognize the efforts of longstanding contributors. Responsibilities for the various community roles fall under these subprojects (repos) as defined by the CloudWeGo team.
+ <table board='4' class='table table-bordered table-striped'>
+      <thead>
+      <tr>
+        <th align='left' style='width:40px;'>Role</th>
+        <th align='left' style='width:40px;'>Responsibilities</th>
+        <th align='left' style='width:40px;'>Requirements</th>
+        <th align='left' style='width:40px;'>Defined by</th>
+      </tr>
+      <tr>
+        <td align='left'>Member</td>
+        <td>Active contributor in the community.</td>
+        <td>Sponsorhips by two approvers or maintainers, and multiple code contributions to the project.</td>
+        <td>GitHub org member</td>
+      </tr>
+      <tr>
+        <td align='left'>Committer</td>
+        <td>Active code contributions and/or issue replies in the subproject.</td>
+        <td>Sponsorhips by two approvers or maintainers and multiple code contributions to the project.</td>
+        <td>GitHub subproject committer team</td>
+      </tr>
+      <tr>
+        <td align='left'>Reviewer</td>
+        <td>Review contributions from other members, and give feedback and guidance.</td>
+        <td>Continuous history of review and authorship in a subproject.</td>
+        <td>GitHub subproject committer team</td>
+      </tr>
+      <tr>
+        <td align='left'>Approver</td>
+        <td>Approve contributions received by the project.</td>
+        <td>Highly experienced, active reviewer and contributor to a subproject.</td>
+        <td>GitHub subproject committer team.</td>
+      </tr>
+      <tr>
+        <td align='left'>Maintainer</td>
+        <td>Set the direction and priorities for a subproject.</td>
+        <td>Demonstrated sense of responsibility and excellent technical judgement for the subproject.</td>
+        <td>GitHub subproject committer team</td>
+      </tr>
```

Comment:
Those are not correct. Please be more careful when do the copying from other places. 
Reviewer -> GitHub subproject reviewer Team
Approver -> GitHub subproject approver Team
Maintainer -> GitHub subproject maintainer Team

Reply:
refer : https://github.com/cloudwego/community/blob/main/COMMUNITY_MEMBERSHIP.md